mne/decoding/ems.py

```diff
@@ -68,7 +68,8 @@ def compute_ems(epochs, conditions=None, picks=None, verbose=None, n_jobs=1):
     else:
         epochs = epochs[conditions]
 
-    epochs.drop_bad_epochs()
+    if epochs._bad_dropped is False:
```

Comment:
this machinery of accessing private attributes should stay internal to epochs instance. Exposing the externals of objects in bad bad bad